webknossos/webknossos/utils.py

```diff
@@ -562,17 +562,23 @@ def enrich_path(path: str | PathLike | UPath, dataset_path: UPath) -> UPath:
     return resolve_if_fs_path(upath)
 
 
+def _ensure_trailing_slash(path: str) -> str:
+    if not path.endswith("/"):
+        return path + "/"
+    return path
+
+
 def dump_path(path: UPath, dataset_path: UPath | None) -> str:
     if is_fs_path(path) and not path.is_absolute():
         if dataset_path is None:
             raise ValueError("dataset_path must be provided when path is not absolute.")
         path = dataset_path / path
```

Comment:
Other edge cases that come to mind: What happens if dataset_path also is not absolute? what if “..” is contained?